sql/hive/src/test/scala/org/apache/spark/sql/hive/parquetSuites.scala

```diff
@@ -425,6 +425,28 @@ class ParquetMetastoreSuite extends ParquetPartitioningTest {
     }
   }
 
+  test("SPARK-15968: nonempty partitioned metastore Parquet table lookup should use cached " +
```

Comment:
I looked in `CachedTableSuite`. I'm not sure that's a good place for this kind of test. That test suite seems focused on testing tables cached by the [CacheManager](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala). This patch is focused on table caching in [HiveMetastoreCatalog](https://github.com/apache/spark/blob/master/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveMetastoreCatalog.scala).

It's difficult to find the best place for these kinds of caching tests. I chose this file because it already had some of these tests. Perhaps [HiveMetastoreCatalogSuite](https://github.com/apache/spark/blob/master/sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveMetastoreCatalogSuite.scala) would be a good candidate for an alternative?